execute/aggregate_test.go

```diff
@@ -171,6 +170,224 @@ func TestAggregate_Process(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:   "empty table",
```

Comment:
Can you name this something different to indicate that there is a table with data in here? Or remove the first table. I looked at this originally and got confused why the table wasn't empty, but I see now there are two tables. I have the same comment for the other tests too.